src/components/Buttons/Buttons.css

```diff
@@ -0,0 +1,22 @@
+.orange-button {
+  width: 13.625em;
+  height: 4em;
+  background: #ffa001;
+  border-radius: 2.438em;
+  font-family: Poppins;
+  font-style: normal;
+  font-weight: normal;
+  font-size: 1.5em;
+  line-height: 2.25 em;
+  color: #ffffff;
+  border: 0px;
+}
+
+.orange-button:active {
+  transform: translateY(0.5px);
+  filter: saturate(200%);
+}
+
+.orange-button:hover {
+  background-color: #ee9804;
+}
```

Comment:
I like the color choice for the hover state! Could we also add `cursor:pointer` to the hover state?

Reply:
Excellent! Added, and thanks!